pandas/core/internals/construction.py

```diff
@@ -903,7 +903,7 @@ def _list_of_dict_to_arrays(
 
     # assure that they are of the base dict class and not of derived
     # classes
-    data = [d if type(d) is dict else dict(d) for d in data]
+    data = [d if type(d) is dict else dict(d) for d in data]  # noqa: E721
```

Comment:
Instead of ignoring the issue, maybe try `[d if isinstance(d, dict) ...]`?

Reply:
@twoertwein yes, sir. So, I guess there is nothing I need to change?